post-build/action.yml

```diff
@@ -87,9 +87,9 @@ runs:
 
     - name: Upload bottles
       if: always() && fromJson(inputs.upload-bottles) && (steps.bottles.outputs.count > 0 || steps.bottles.outputs.skipped > 0)
-      uses: actions/upload-artifact@v3-node20
+      uses: actions/upload-artifact@v4
       with:
-        name: bottles
+        name: bottles-${{ inputs.runner }}
```

Comment:
This conflicts with the failed bottles name above

Reply:
Ah yes, will change the above.

Reply:
Actually, on second thoughts, this probably means we'd have problems during the migration because the failed bottles uploaded before the change might be accidentally picked up.

We'll need to either choose a new prefix, or delete the existing artifacts for failed bottles.